package.json

```diff
@@ -79,9 +80,5 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
-  },
-  "publishConfig": {
```

Comment:
a bit lost here, why do we remove it now?

Reply:
Because Lerna ignore that thing at all — since it publishes the packages singularly looking at the package.json of the packages — not the main one.